### PR TITLE
Feature/labels

### PR DESCRIPTION
**Pull request by kami3la**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 - Logo Component.
+- Labels Component.
```
Comment (Mrozelek) [inline CHANGELOG.md:11]: Why is it plural?

File: src/components/labels/labels.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.labels {
```
Comment (Mrozelek) [inline src/components/labels/labels.scss:1]: Why is it plural?

File: src/components/labels/labels.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import './labels.scss';
+import WebComponent, { createElementFromString } from '../../common/WebComponent';
+
+class LabelsComponent implements WebComponent {
```
Comment (Mrozelek) [inline src/components/labels/labels.ts:4]: Why is it plural?

File: src/components/labels/labels.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  color: #fff;
+  display: inline-block;
+  font-size: 1.5rem;
+  font-weight: 500;
+  outline: none;
+  padding: 0.6rem 1.6rem;
+  text-align: center;
+  width: auto;
+
+  &__settings-style {
```
Comment (Mrozelek) [inline src/components/labels/labels.scss:14]: Is there a reason to call this style `style`? We all know it is indeed a style.

File: src/components/labels/labels.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  private componentElement: string = '';
```
Comment (Mrozelek) [inline src/components/labels/labels.ts:5]: You didn't use `yarn lint` command.

File: src/components/labels/labels.ts [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+  boardWhoPlays(name: string): string {
+    this.componentElement = `${name} plays!`;
+    return this.componentElement;
+  }
+
+  boardChooseCategory(): string {
+    this.componentElement = 'choose category';
+    return this.componentElement;
```
Comment (Mrozelek) [inline src/components/labels/labels.ts:32]: Just use the constructor to specify the text in label. What if I would like to have label with 20 different texts? Would I create a method for each text?

**Review by Mrozelek — CHANGES_REQUESTED** [review]

**Review by adax10 — APPROVED** [review]

**Review by Mrozelek — APPROVED** [review]

**Review by brzeczkowskaw — APPROVED** [review]

**Pull request merged**